### PR TITLE
Misc tweaks and fixes - July 2020

### PR DESCRIPTION
**Pull request by RecursiveHook**
- "Fixes" missing OG roles for add/remove ops
- OG List updates
- Fix ARF to remove any higher-level friend states

### CODE REVIEW

File: cogs/autorole.py [hunk 1]
```diff
@@ -551,7 +551,11 @@ async def promote_to_friend(self, ctx, *users: str):
                 'ghost-proxy-friend'
             ],
             users,
-            role_limits=['ghost-proxy-member']
+            role_limits=[
+                'ghost-proxy-member',
+                'ghost-proxy-legacy',
+                'ghost-proxy-envoy'
+            ]
```
Comment (OxyOCE) [inline cogs/autorole.py:558]: Good catch, I don't think we need this for standard procedure but makes mistakes easier to fix.

File: cogs/config/roles.py [hunk 2]
```diff
@@ -7,8 +7,8 @@
             },
             'other_games': {  # Based on User Input
                 'og-ac': ['animal crossing', 'ac', 'animal crossing new horizons'],
-                # 'og-ant': ['anthem', 'ant'],
-                # 'og-apex': ['apex legends', 'apex', 'apex: legends'],
+                'og-ant': ['anthem', 'ant'],
+                'og-apex': ['apex legends', 'apex', 'apex: legends'],
```
Comment (OxyOCE) [inline cogs/config/roles.py:11]: Sweet as, I'm sure we'll fix this properly in the autorole refactor.

File: cogs/gaminator.py [hunk 3]
```diff
@@ -122,7 +122,7 @@ class Gaminator(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
 
-    @commands.command(name='other-games')
+    @commands.command(name='other-games', aliases=['og'])
```
Comment (OxyOCE) [inline cogs/gaminator.py:125]: Ah yep, to be honest 'other-games' was a placeholder but if you're happy with the name, I'm happy.

**Comment by RecursiveHook** [comment]
Awesome, merging, thanks Oxy!

**Pull request merged**